clients/client/src/main/java/org/projectnessie/client/api/CreateNamespaceBuilder.java

```diff
@@ -26,5 +27,7 @@
  */
 public interface CreateNamespaceBuilder extends OnNamespaceBuilder<CreateNamespaceBuilder> {
 
+  CreateNamespaceBuilder properties(Map<String, String> properties);
```

Comment:
Would be nice to have a `property(String, Streing)` method here as well.
Should be easy to implement to add a field initialized with `ImmutableNamespaceUpdate.builder()` in `HttpCreateNamesapce`

Reply:
done